src/internal/connector/exchange/delta_get.go

```diff
@@ -4,9 +4,9 @@ import (
 	"context"
```

Comment:
We can actually just delete this entire file now that we're upgrading to a version of graph SDK that supports the `select` clause on delta queries. The overall code can be made to look like other calls to graph SDK that we make

All of the code from #1643 can be removed and we can just use the graph SDK types for the query options and the regular delta call for sending requests. We should still iterate through results manually though so `service_iterators.go` will have something like
```go
builder := gs.Client().
  UsersById(user).
  MailFoldersById(directoryID).
  Messages().
  Delta()

// Lower down when we need to actually send the request
resp, err := builder.Get(ctx, options)
```

Contacts should also be updated to just use the graph SDK calls instead of the calls in this file